fenix/app/src/main/java/org/mozilla/fenix/compose/Menu.kt

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.compose
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.DropdownMenu
+import androidx.compose.material.DropdownMenuItem
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.platform.LocalConfiguration
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Popup action menu.
+ *
+ * @param showMenu Whether or not the menu is currently displayed to the user.
+ * @param onDismissRequest To be called when user requests to dismiss the menu (on rotation change,
+ * on outside click, on back pressed, on interacting with any item).
+ * @param menuItems List of items to be displayed in the menu.
+ * @param menuItems List of items to be displayed in the menu.
+ */
+@Composable
+fun Menu(
+    showMenu: Boolean,
+    onDismissRequest: () -> Unit,
+    menuItems: List<MenuItem>,
+    modifier: Modifier = Modifier,
+) {
+    DisposableEffect(LocalConfiguration.current.orientation) {
+        onDispose { onDismissRequest() }
+    }
+
+    MaterialTheme(shapes = MaterialTheme.shapes.copy(medium = RoundedCornerShape(5.dp))) {
```

Comment:
```suggestion
    MaterialTheme(shapes = MaterialTheme.shapes.copy(medium = RoundedCornerShape(5.dp))) {
```
Is this really 5dp? I didn't check the figma but it's always odd to me to see a odd number.

Reply:
I'm more curious about the purpose of wrapping this Composable with `MaterialTheme`. Is that the only way to customize the shape?

Reply:
This is a small detail so I don't want to hold anything up too long. @gabrielluong, since we don't have shapes figured out yet for Fenix, what's your opinion for this particular case of whether to use `MaterialTheme` vs `Modifier.clip()`?

Reply:
Let's file a followup about investigating using `Modifier.clip(shape)`. I don't know enough about it to really make a call so it's just a coin flip.

Reply:
Filed [1826118](https://bugzilla.mozilla.org/show_bug.cgi?id=1826118) for this.